moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/parameterization/work_space/pose_model_state_space.h

```diff
@@ -145,5 +145,6 @@ class PoseModelStateSpace : public ModelBasedStateSpace
   };
 
   std::vector<PoseComponent> poses_;
+  double jump_factor_;
```

Comment:
Would be nice to have a docstring describing this.

Reply:
I just restored this code from #2910.